web-wallet/package-lock.json

```diff
@@ -62,6 +62,11 @@
         "node": ">=20.0.0"
       }
     },
+    "../../../lamb-types": {
+      "version": "0.61.9",
+      "extraneous": true,
+      "license": "MIT"
+    },
```

Comment:
Should we have this?

Reply:
No: probably forgot to unlink it while local testing.
Won't do harm, but it's better to remove it: thanks for noticing.